### PR TITLE
Remove fastest-json-copy dependency from PropertyDDS

### PR DESCRIPTION
**Pull request by Abe27342**
## Description

`fastest-json-copy` is generally unmaintained and has an open CVE for prototype pollution. This partially reverts the performance improvements made in #8141 by moving back to lodash (an existing dependency of propertyDDS) for `cloneDeep`.

### CODE REVIEW

**Review by ashishswami22 — COMMENTED** [review]
Looks good. Thank you.

**Review by ashishswami22 — COMMENTED** [review]
Can we please prioritize this for the next GA.

**Comment by Abe27342** [comment]
> Can we please prioritize this for the next GA.

I'm updating this PR today to get CI passing on it and will get a review shortly thereafter.

**Review by CraigMacomber — APPROVED** [review]

File: experimental/PropertyDDS/packages/property-dds/src/propertyTree.ts [hunk 1]
```diff
@@ -377,7 +376,9 @@ export class SharedPropertyTree extends SharedObject {
 			message.type === MessageType.Operation &&
 			message.sequenceNumber > this.skipSequenceNumber
 		) {
-			const change: IPropertyTreeMessage = this.decodeMessage(cloneDeep(message.contents));
+			const change: IPropertyTreeMessage = this.decodeMessage(
```
Comment (Abe27342) [inline experimental/PropertyDDS/packages/property-dds/src/propertyTree.ts:379]: the typing on lodash is evidently more strict than on fastest-json-copy, hence this cast (`message.contents` has type `unknown` and lodash's typing is the expected `cloneDeep: <T>(x: T): T`)

**Pull request merged**